Content rekenen met breuken en kommagetallen (#41)
florisdf/hoe-zit-het: content/lessen/wiskunde/rekenen_rationaal/rekenen_gehele_getallen/index.md

---
title: "Rekenen met gehele getallen"
date: 2020-06-21T15:00:00+02:00
weight: 1
draft: true
images: []
categories: ["wiskunde", "getallen", "1e middelbaar"]
tags: ["Optellen", "Aftrekken", "Vermenigvuldigen", "Delen", "Rekenen", "Bewerkingen"]
description: "Deze les is een korte herhaling van de basisbewerkingen: optellen,
aftrekken, vermenigvuldigen en delen. We bekijken aan de hand van voorbeelden nog eens
snel hoe dit precies werkt."
images: ['/lessen/wiskunde/rekenen_rationaal/rekenen_gehele_getallen/img/optelling.png', '/lessen/wiskunde/rekenen_rationaal/rekenen_gehele_getallen/img/aftrekking.png',
'/lessen/wiskunde/rekenen_rationaal/rekenen_gehele_getallen/img/vermenigvuldiging.png', '/lessen/wiskunde/rekenen_rationaal/rekenen_gehele_getallen/img/deling.png',
'/lessen/wiskunde/rekenen_rationaal/rekenen_gehele_getallen/img/optelling_met_negatieve_getallen.png']
---

> Om moeilijke berekeningen succesvol op te lossen, is het belangrijk dat we kunnen **rekenen met gehele getallen**. Moeilijkere bewerkingen en concepten, 

Reviewer: ```suggestion
Om later moeilijke berekeningen succesvol op te kunnen lossen, is het belangrijk dat we eerst leren **rekenen met gehele getallen**. Moeilijkere bewerkingen en concepten, 
```

zoals machten en vierkantswortels, bouwen hier namelijk op voort. We kunnen ook zeggen dat we werken in $\mathbb{Z}$,

> de verzameling van de gehele getallen.

Reviewer: ```suggestion
zoals machten en vierkantswortels, bouwen namelijk voort op de bewerkingen van gehele getallen. Als symbool voor "de verzameling van de gehele getallen" gebruiken we $\mathbb{Z}$. Daarom zeggen we ook wel "rekenen in $\mathbb{Z}$" in plaats van "rekenen met gehele getallen".
```

## Gehele getallen bij elkaar optellen

> 

Reviewer: ```suggestion

De onderdelen van een optelling noemen we de **termen** van de optelling. Tussen die termen noteren we een **plusteken** en de uitkomst van de optelling is de **som**.

```

{{< svg "img/optelling.svg" "Een optelling bestaat uit termen die als resultaat een som geven. We gebruiken het plusteken." >}}

Een optelling kunnen we visueel voorstellen op een **getallenas**. Wanneer je **drie sprongen naar rechts** gaat en je daarna nog eens

> **twee sprongen naar rechts** gaat, heb je in totaal **vijf sprongen naar rechts** gezet. We kunnen dit schrijven als de som $3 + 2 = 5$.

Reviewer: ```suggestion
Een optelling kunnen we visueel voorstellen op een **getallenas**. We beginnen op de $0$ van de getallenas en telkens we een optelling tegenkomen, zetten we een aantal stappen. Wanneer je bv. vanaf de $0$ **drie stappen naar rechts** gaat en je daarna nog eens
**twee stappen naar rechts** gaat, kom je uit op de $5$ van de getallenas. We kunnen dit schrijven als de som $3 + 2 = 5$.
```

Reviewer: Ik zou niet zowel "stappen" als "sprongen" gebruiken, maar één van de twee kiezen en dat consistent gebruiken doorheen de les.

{{< svg "img/optelling_getallenas.svg" "De optelling met twee positieve getallen, weergegeven op een getallenas" >}}

> 

Reviewer: ```suggestion

Het kan ook dat een optelling **meer dan twee termen** heeft. In dat geval gaan we gewoon verder met stappen. Stel dat we de volgende optelling moeten uitrekenen:

$$5 + 2 + 4$$

Eerst zetten we vijf stappen naar rechts, vervolgens twee stappen naar rechts en ten slotte vier stappen naar rechts. Dan komen we uit op $11$.
```

Reviewer: Hier eventueel ook nog een tekening met de as toevoegen

> ### Negatieve getallen

Reviewer: Ik zou de secties "Negatieve getallen" en "Wat te doen met verschillende tekens?" samen in een nieuwe paragraaf zetten "Gehele getallen optellen en aftrekken". Die paragraaf komt dan na de paragrafen "Natuurlijke getallen optellen" en "Natuurlijke getallen aftrekken":

1. Natuurlijke getallen optellen: beginnen op $0$ en telkens naar **rechts** stappen.
2. Natuurlijke getallen aftrekken: beginnen op $0$, voor het eerste getal (waar geen minteken voor staat) naar rechts stappen. Bij elke volgende aftrekking naar **links** stappen.
3. Gehele getallen optellen en aftrekken: beginnen op $0$. Na gestapt te hebben, draai je altijd terug naar rechts. Telkens wanneer je een minteken tegenkomt, draai je je om.

We kunnen ook negatieve getallen optellen. Als één van beide termen negatief is, dan trek je dit getal
af van de andere term. Zo krijg je $4$ wanneer je $6$ en $-2$ optelt. Je gaat eerst **zes sprongen naar rechts** en dan **twee sprongen naar links**.

> {{< svg "img/optelling_negatief_getal.svg" "De optelling met een negatief getal, weergegeven op een getallenas." >}}

Reviewer: Ik zou de twee pijltjes uiterst links weglaten. Die lijken me wat verwarrend

Als beide termen negatief zijn, dan tel je de **absolute waarden** van beide getallen op{{<mute " (dat zijn de getallen zonder teken) ">}} en plaats
je er een **minteken** voor. Zo krijg je $-5$ wanneer je $-2$ en $-3$ optelt. $\mid -2\mid  + \mid -3 \mid = 5$, met een minteken voor wordt dit $-5$.

Anders gezegd ga je eerst **twee sprongen naar links** en vervolgens nog eens **drie sprongen naar links**.

{{< svg "img/optelling_negatieve_getallen.svg" "De optelling met twee negatieve getallen, weergegeven op een getallenas." >}}

## Gehele getallen van elkaar aftrekken

> 

Reviewer: ```suggestion

De onderdelen van een aftrekking noemen we, net als bij een optelling, de **termen**. Tussen die termen noteren we een **minteken** en de uitkomst van de aftrekking is het **verschil**.

```

{{< svg "img/aftrekking.svg" "Een aftrekking bestaat uit termen die als resultaat een verschil geven. We gebruiken het minteken." >}}

Een geheel getal aftrekken van een ander geheel getal betekent hetzelfde als zijn tegengestelde erbij optellen. Dus we kunnen
het voorbeeld herschrijven als:

$$10 + (-2) = 8$$

> Merk op dat we $-2$ tussen haakjes hebben geschreven. Dit doen we om verwarring te voorkomen. We tellen het getal $-2$ op bij $10$.

Reviewer: ```suggestion
Merk op dat we $-2$ tussen haakjes hebben geschreven. Dit doen we omdat "$10 + - 2$" nogal verwarrend en slordig is.

$10 - 2$ is dus hetzelfde als $10 + (-2)$, of: "zet tien sprongen naar rechts en twee naar links". Dan komen we inderdaad uit op $8$!
```

Reviewer: Als we de paragraaf "Gehele getallen van elkaar aftrekken" vervangen door "Natuurlijke getallen van elkaar aftrekken", hoeven we nog niet te beginnen over negatieve getallen. Zo blijven de eerste twee paragrafen heel intuïtief terwijl ze vertrouwd raken met _the mental picture_ van het stappen op de getallenas. In de derde paragraaf "Gehele getallen optellen en aftrekken" gaan we die _mental picture_ dan veralgemenen naar positieve én negatieve getallen.

### Wat te doen met verschillende tekens?

> Soms kom je uitdrukking tegens zoals $2 -(-4))$. Wat moet je doen met al deze tekens en hoe bepaal je de uitkomst? Daarvoor kan je je best een persoon
> langs een getallenas voorstellen. 

Reviewer: ```suggestion
Soms kom je uitdrukking tegens zoals $2 -(-4)$. Wat moet je doen met al deze tekens en hoe bepaal je de uitkomst? Daarvoor kunnen we weer terugdenken aan
onze getallenas. 
```

> Een **cijfer** vertelt ons het **aantal stappen** dat deze persoon zet. De richting {{< mute "(links of rechts)" >}} wordt bepaald
> door het teken. We starten standaard naar rechts, zoals we ook van links naar rechts schrijven. Als we een **plusteken** tegenkomen in de uitdrukking, dan gaan we
> verder **in dezelfde richting**. Komen we echter een **minteken** tegen, dan draaien we om een gaan we **in de andere richting** verder.

Reviewer: ```suggestion
De **getallen** gaan ons weer vertellen **hoeveel stappen** we moeten zetten. De **richting** {{< mute "(naar links of naar rechts)" >}} waarin we moeten stappen, wordt bepaald
door de tekens die we tegenkomen. We starten altijd op de $0$ en naar rechts, net zoals we ook van links naar rechts schrijven. Telkens wanneer we een **plusteken** tegenkomen in de uitdrukking, dan gaan we
verder **in dezelfde richting**. Komen we echter een **minteken** tegen, dan draaien we om en gaan we **in de andere richting** verder.
```

{{< svg "img/verschillende_tekens.svg" "Een persoon die langs de getallenas beweegt." >}}

> De persoon die de bewerking hierboven uitvoert langs de getallenas begint dus met **twee stappen naar rechts** te zetten. Vervolgens draait hij zich om, waarna hij zich
> nogmaals omdraait en dan **vier stappen (naar rechts)** zet.

Reviewer: ```suggestion
Bij $2-(-4)$ beginnen we dus met **twee stappen naar rechts** te zetten. Vervolgens komen we na de $2$ een minteken tegen en moeten we ons dus omdraaien. Maar dan komen we *opnieuw* een minteken tegen (dat van de $-4$) waardoor we
een tweede keer moeten omdraaien. We kijken dus terug naar rechts voordat we onze **vier stappen** zetten. Na die vier stappen, komen we uit bij $6$.
```

Reviewer: Ik zou de figuur misschien opdelen in 4 deel-afbeeldingen:

1. Twee stappen zetten $**2** - (- 4)$
2. Naar links draaien $2 **-** (- 4)$
3. Naar rechts draaien $2 - (**-** 4)$
4. Vier stappen zetten $2 - (- **4**)$

Bij elk deel zou ik in de illustratie de "2 - (- 4)" zetten en de delen zwart houden die dan worden uitgevoerd. De rest van de berekening grijs.

## Gehele getallen vermenigvuldigen

> 

Reviewer: ```suggestion

De onderdelen van een vermenigvuldiging noemen we de **factoren**. Tussen de factoren noteren we een **maalteken** en de uitkomst van de vermenigvuldiging is het **product**.

```

{{< svg "img/vermenigvuldiging.svg" "Een vermenigvuldiging bestaat uit factoren die als resultaat een product geven. We gebruiken het maalteken." >}}

> Een vermenigvuldiging is eigenlijk een manier om een **optelling van dezelfde getallen korter** te schrijven.

Reviewer: ```suggestion
Een vermenigvuldiging is een manier om een **optelling van eenzelfde getal korter** te schrijven.
```

Reviewer: Ik zou hier ook weer een getallenas gebruiken waarbij we voor $3 \cdot 2$ drie keer twee stappen (naar rechts) zetten. Dan zou ik uitbreiden naar vermenigvuldiging met meerdere factoren, bv. $5 \cdot 3\cdot 2$. Dan gaan we vijf keer onze "drie keer twee stappen" zetten. Hier kunnen we ook tonen dat wanneer we $3 \cdot 5 \cdot 2$ uitrekenen, we op dezelfde plek uitkomen aangezien de vermenigvuldiging van gehele getallen _commutatief_ is.

Vervolgens zou ik bespreken wat we met mintekens moeten doen. Bv. $3 \cdot (-2)$. Dat betekent dat we drie keer twee _omgekeerde_ stappen moeten zetten. $-3 \cdot 2$ is dan eerst omdraaien en vervolgens drie keer twee stappen zetten. Zo zien ze dat $3 \cdot (-2) = -3 \cdot 2$ en dat we de mintekens in een vermenigvuldiging dus voorop mogen zetten. Dat betekent dat $-3 \cdot (-2) = - ( - (3 \cdot 2))$ en we ons dus twee keer omdraaien om vervolgens drie keer twee stappen te zetten. We komen dus uit op $6$. Omdat twee keer omdraaien hetzelfde resultaat heeft als niet omdraaien, kunnen we zeggen dat $- ( - (3 \cdot 2)) = 3 \cdot 2$.

Dat naar voor schuiven van mintekens doen we dan nog eens voor $5 \cdot (-3) \cdot 2 = - (5 \cdot 3 \cdot 2) = - 30$. Dan ook nog eens voor $5 \cdot (-3) \cdot (-2) = - (- (5 \cdot 3 \cdot 2) ) = 30$. En ten slotte nog eens voor $-5 \cdot (-3) \cdot (-2) = - ( - (- (5 \cdot 3 \cdot 2) ) ) = -30$. We zien dat een **oneven** aantal mintekens hetzelfde resultaat geeft als **een minteken** en dat een **even** aantal mintekens hetzelfde resultaat geeft als **een plusteken**. Als je twee keer, vier keer, zes keer enz. omdraait, eindig je in dezelfde richting. Als je één keer, drie keer, vijf keer enz. omdraait, eindig je in de omgekeerde richting.

Reviewer: Of we kunnen zeggen dat we een vermenigvuldiging kunnen zien als grotere stappen zetten dan normaal. Normaal zetten we altijd een stap van $1$, bij een vermenigvuldiging kan dat iets anders zijn. Bij $6 \cdot 3$, bijvoorbeeld, zetten we zes stappen met stappen die drie keer groter zijn dan normaal.

$$10 \cdot \orange{2} = \underbrace{10 + 10}_{\orange{2}\ \text{keer}} = 20$$

> Welk teken denk je dat $3 \cdot (-2)$ zal hebben? Als je dit uitschrijft zoals hierboven, kan je dit makkelijk te weten komen!

Reviewer: ```suggestion
Een eenvoudige vermenigvuldiging als $10 \cdot 2$ heb je wellicht al enkele jaren geleden leren oplossen. Welk teken denk je dat $3 \cdot (-2)$ zal hebben? Als je dit uitschrijft zoals hierboven, kan je dit snel te weten komen!
```

$$\orange{3} \cdot (-2) = \underbrace{(-2) + (-2) + (-2)}_{\orange{3}\ \text{keer}} = -6$$

Ook een vermenigvuldiging met **twee negatieve factoren** kan je op deze manier schrijven. Je moet de getallen dan niet optellen,
maar **aftrekken**.

$$\orange{-3} \cdot (-4) = \underbrace{\green{-}(-4) - (-4) - (-4)}_{\orange{-3}\ \text{keer}} = 12$$

Nu vraag je je misschien af vanwaar de $\green{-}$ aan het begin komt. Eigenlijk kunnen we steeds een teken schrijven voor de eerst term,
maar bij de eerste twee voorbeelden maakt dit geen verschil. 

$$10 \cdot \orange{2} = \underbrace{\green{+}10 + 10}_{\orange{2}\ \text{keer}} = 20$$

$$\orange{3} \cdot (-2) = \underbrace{\green{+}(-2) + (-2) + (-2)}_{\orange{3}\ \text{keer}} = -6$$

### Negatieve getallen

Hieruit kunnen we besluiten dat als er een **even aantal negatieve factoren** zijn,
de uitkomst **positief** is. Als er een **oneven aantal negatieve factoren** zijn, dan is de uitkomst **negatief**.

> Nog niet overtuigd :thinking_face:? Bekijk de alinea *Wat te doen met verschillende tekens?* nog eens grondig!

## Gehele getallen delen

> 

Reviewer: ```suggestion

Een deling bestaat uit een **deeltal** die we moeten delen door een **deler**. Het resultaat van een deling noemen we het **quotiënt**.

```

{{< svg "img/deling.svg" "Een deling bestaat uit een deeltal en een deler die als resultaat een quotiënt geven. We gebruiken het deelteken." >}}

> Een deling is eigenlijk het omgekeerde van een vermenigvuldiging.

Reviewer: Hier de getallenas ook weer bij betrekken. We zouden kunnen zeggen dat we een vermenigvuldiging ook kunnen zien als grotere stappen zetten. $6 \cdot 3$ is zes stappen zetten met stappen die drie keer groter zijn dan normaal. Een deling is het omgekeerde waarbij we bij $6 : 3$ zes stappen zetten met stappen die drie keer kleiner zijn dan normaal.

Reviewer: Nog niet 100% zeker of deze analogie wel zo nuttig is.

Author: Ja ik had ook al aan zoiets gedacht, met een zeker basis (uit de lineaire algebra), maar dit kan misschien voor velen verwarrender zijn dan dat het gaat helpen.
